### PR TITLE
Updates ReadMe with examples for Version

### PR DESCRIPTION
**Pull request by arunchandarQA**

### CODE REVIEW

**Comment by arunchandarQA** [comment]
@holytshirt Could you merge this change in ReadMe.

**Comment by holytshirt** [comment]
Done, thanks!

File: README.md [hunk 1]
```diff
@@ -120,6 +120,54 @@ octopose generate --ignore Tasks
 ```
 This will remove projects from the manfest.
 
+
+### Generate a manifest for a specific project and version
+```
+octopose generate -p Huddle.ABC -v '{\"Huddle.ABC\" : \"1.99.9.999\"}'
```
Comment (domgreen) [inline README.md:126]: should probably keep examples to be using semver to promote best practice

Comment (holytshirt) [reply README.md:126]: @domgreen so you don't like the new 2020.05.02.345 format?

**Pull request merged**